static/js/zamboni/devhub.js

```diff
@@ -885,8 +885,8 @@ function initCatFields(delegate) {
             $("input", $main).attr("checked", false).attr("disabled", false);
         };
         checkMainDefault();
-        $('input', $main).live('change', checkMain);
-        $('input', $misc).live('change', checkOther);
+        $('input', $main).on('change', checkMain);
```

Comment:
These side-step the recommended approach for replacement to live to retain the use of the context (2nd arg to `$()`).